docs/docs/getting-started.md

```diff
@@ -45,6 +45,15 @@ $ webpack
 >
 > If you are using ES2015, you will want to also use the `babel-preset-es2015` package.
 
+**Note:** by default, React will be in development mode, which is slower, and not advised for production. To use React in production mode, set the environment variable `NODE_ENV` to `production` (using envify or webpack's DefinePlugin). For example:
+
+```js
+new webpack.DefinePlugin({
+  "process.env": {
+    NODE_ENV: JSON.stringify("production")
```

Comment:
I think it shouldn't be an issue here because this fragment is not a complete config, and it has enough context around it.